Formula/yarn.rb

```diff
@@ -3,8 +3,8 @@
 class Yarn < Formula
   desc "Javascript package manager"
   homepage "https://yarnpkg.com/"
-  url "https://registry.npmjs.org/yarn/-/yarn-0.15.1.tgz"
-  sha256 "f99fd587e84987909d5f9e918b8fe524349fdc548e5bc5c380c8f8c0a70c6b87"
+  url "https://yarnpkg.com/downloads/0.15.0/yarn-v0.15.0.tar.gz"
```

Comment:
Any objections to making this https://github.com/yarnpkg/yarn/releases/download/v0.15.0/yarn-v0.15.0.tar.gz? It'll make it easier for us to query for new versions. Also, as @zmwangx pointed out this needs to be 0.15.1. It may be worth using the auto-generated GitHub tarball from the tag if that's going to make that easier: https://github.com/yarnpkg/yarn/archive/v0.15.1.tar.gz